### PR TITLE
fix(node): use old "process.nextTick" polyfill if Deno.core is not available

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Comment by bartlomieju** [comment]
Screenshot 2021-11-17 at 21 18 05

File: node/_next_tick.ts [hunk 1]
```diff
@@ -1,14 +1,13 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 
+// deno-lint-ignore-file no-inner-declarations
```
Comment (dsherret) [inline node/_next_tick.ts:3]: Perhaps we should still keep the declarations at the top level, but just make certain statements conditional?

**Review by dsherret — APPROVED** [review]
LGTM

File: node/_next_tick.ts (hunk 1 above)
Comment (bartlomieju) [reply node/_next_tick.ts:3]: I'd prefer not - I want to save a reference to `Deno.core` in `core` constant and reuse that inside functions.

**Pull request merged**